### ISSUE TITLE
[NPUB31647] J-Stars Victory VS+ - Game freezing with audio output.

### ISSUE THREAD

**Issue by JohnLoveJoy** [RPCS3/rpcs3#14506]
### Quick summary

After a few minutes of gameplay in any battle mode, the game just freezes, but the audio keeps playing. Looking at the log always comes the message:

 "E VM: [SPU[0x5000100] SP_LIB CellSpursKernel5] Access violation writing location 0xffdead00 (unmapped memory)
E VM: [SPU[0x2000100] SP_LIB CellSpursKernel2] Access violation writing location 0xffdead00 (unmapped memory)"

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/12384208/RPCS3.log.gz)

### Attach capture files for visual issues

![PS0002](https://github.com/RPCS3/rpcs3/assets/31490343/5b20a7ef-2ae4-4bd0-9f7a-588aff22c6f2)
![PS0001](https://github.com/RPCS3/rpcs3/assets/31490343/22560e6d-9236-4baa-87b4-feff780d4cdc)

### System configuration

Windows 10 - Updated.
RPCS3 0.0.29-15497; ASMJIT, Vulkan, Native Res, Safe block size

CPU Ryzen 5500U (Zen2) 6/12 
GPU - Vega 7

**Comment by AniLeo** [comment]
As it is, it's an invalid report

Using many settings not listed in the wiki page without any explanation as to why they need to be changed from defaults, and RPCS3 is inside OneDrive folder

**Comment by StevenMiller123** [comment]
This is a duplicate of an issue I made.
https://github.com/RPCS3/rpcs3/issues/7858

**Comment by JohnLoveJoy** [comment]
> As it is, it's an invalid report
> 
> Using many settings not listed in the wiki page without any explanation as to why they need to be changed from defaults, and RPCS3 is inside OneDrive folder


The issue happens with any settings, including the default ones. The fact that I have different configurations in the log is just a vestige that I spent some time investigating with all possible configurations, then when I got tired, I came to post here. I think the folder isn't relevant as it's not on OneDrive, it was created on the desktop, it's just how windows names things I guess(?).

![P3000](https://github.com/RPCS3/rpcs3/assets/31490343/18c11263-bc8c-46a1-a43c-d7b1e22d729f)
![P3001](https://github.com/RPCS3/rpcs3/assets/31490343/2d02ce26-0fe5-4e39-98df-0cb7a59e9211)
